src/main/scala/org/broadinstitute/dsde/firecloud/service/ServiceUtils.scala

```diff
@@ -0,0 +1,67 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import akka.actor.ActorContext
+import akka.event.LoggingAdapter
+import spray.client.pipelining._
+import spray.http.HttpHeaders.{Cookie, Origin, RawHeader}
+import spray.http.StatusCodes._
+import spray.http.{HttpRequest, HttpResponse, RequestProcessingException, StatusCodes}
+import spray.routing.RequestContext
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.reflect._
+import scala.util.{Failure, Success}
+
+object ServiceUtils {
+
+  def listEntities(
+      log: LoggingAdapter,
+      context: ActorContext,
+      url: String,
```

Comment:
instead of passing all these arguments into the method, either create a new case class with those args as members and pass the case class, OR make those args members of ServiceUtils, then instantiate a new ServiceUtils when you want to run it.